river/command/tags.zig

```diff
@@ -102,6 +104,28 @@ pub fn toggleViewTags(
     }
 }
 
+/// Switch focus to tags that were selected before
+/// If there are no remembered tags, do nothing
+pub fn lastFocusedTags(
+    allocator: *std.mem.Allocator,
+    seat: *Seat,
+    args: []const []const u8,
+    out: *?[]const u8,
+) Error!void {
+    const last_tags = seat.focused_output.pending.last_tags;
+    if (last_tags == 0) {
+        out.* = try std.fmt.allocPrint(allocator, "no remembered tags found on this output", .{});
+        return Error.Other;
+    }
```

Comment:
Instead of having this special case, why not just initialize `last_tags` to 1 and assert that it is `!= 0` here?

Reply:
You are absolutelly right, I got stuck in bspwm world where was not always the same. Will commit update, it should simplify it a lot